src/clj/muuntaja/format/msgpack.clj

```diff
@@ -1,3 +1,4 @@
+;; [clojure-msgpack "1.2.0" :exclusions [org.clojure/clojure]]
```

Comment:
Putting this to the module docstring with explanation would be nice, then it would show up in the API docs once we have them.